src/modules/follows/FollowService.ts

```diff
@@ -0,0 +1,227 @@
+		if (queryResponse.success && queryResponse.data[0]) {
+			// field_name based on the RETURN in the query
+			const numOfPath = neo4j.integer.toNumber(queryResponse.data[0].get("numOfPath"));
+			const relId = neo4j.integer.toNumber(queryResponse.data[0].get("relId"));
+			return {
+				success: true,
+				message: queryResponse.message,
+				data: { numOfPath: numOfPath, relId: relId },
+			};
+		} else {
+			return queryResponse;
+		}
+	}
+
+	/**
+	 * Verify if there is a `Follows` relationship between src and target user
+	 * @param srcUserId
+	 * @param targetUserId
+	 * @returns true if there is, otherwise, false
+	 */
+	async verifyRelFollows(srcUserId: UserInfo["userId"], targetUserId: UserInfo["userId"]) {
+		const query = followQueries.GET_RELATIONSHION_BETWEEN_USERS;
+		const params = {
+			src: srcUserId,
+			target: targetUserId,
+		};
+		const { success, data } = await neo4jInstance.runQueryInTransaction(
+			query,
+			params,
+			QueryMode.read
+		);
+
+		const relExists = success && data != undefined && data[0].get("relExists") == true;
+		return relExists;
+	}
+
+	/**
+	 * Get the list of account_ids the user follows
+	 * @param userId
+	 * @returns
+	 */
+	async getFollows(userId: UserInfo["userId"]) {
+		const query = followQueries.GET_FOLLOWS_FOR_USER;
+		const params = {
+			userId: userId,
+		};
+		const queryResponse = await neo4jInstance.runQueryInTransaction(query, params, QueryMode.read);
+		if (queryResponse.success && queryResponse.data.length > 0) {
+			// update the return data
+			const listOfUserIds = [];
+			// will receive an array of IDs which the user follows
+			for (const record of queryResponse.data) {
+				listOfUserIds.push(record.get("followId"));
+			}
+
+			return {
+				success: true,
+				message: queryResponse.message,
+				data: listOfUserIds,
+			};
+		} else {
+			return queryResponse;
+		}
+	}
+
+	/**
+	 * Get the list of account_ids who follow this user
+	 * @param userId
+	 * @returns
+	 */
+	async getFollowers(userId: UserInfo["userId"]) {
+		const query = followQueries.GET_FOLLOWERS_FOR_USER;
+		const params = {
+			userId: userId,
+		};
+		const queryResponse = await neo4jInstance.runQueryInTransaction(query, params, QueryMode.read);
+		if (queryResponse.success && queryResponse.data.length > 0) {
+			// update the return data
+			const listOfUserIds = [];
+			// will receive an array of IDs which follows the user
+			for (const record of queryResponse.data) {
+				listOfUserIds.push(record.get("followerId"));
+			}
+
+			return {
+				success: true,
+				message: queryResponse.message,
+				data: listOfUserIds,
+			};
+		} else {
+			return queryResponse;
+		}
+	}
+
+	/**
+	 * Unfollow a user
+	 * @param srcUserId ID of the actor
+	 * @param targetUserId ID of the target
+	 * @returns Returns error if `Follows` already exists
+	 */
+	async unFollow(srcUserId: UserInfo["userId"], targetUserId: UserInfo["userId"]) {
+		return await this.deleteRelFollows(srcUserId, targetUserId);
```

Comment:
await is unnecessary

```suggestion
		return this.deleteRelFollows(srcUserId, targetUserId);
```